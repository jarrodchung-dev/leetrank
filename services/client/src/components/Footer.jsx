import React from "react";

const Footer = (props) => {
  return (
    <footer className="footer is-light is-fixed-bottom">
      <div className="container">
        <div className="columns is-multiline">
          <div className="column is-4">
            <span>Copyright Â© 2020 Jarrod Chung</span>
          </div>
          <div className="column is-4">
            <div className="row"></div>
          </div>
          <div className="column is-4"></div>
        </div>
      </div>
    </footer>
  );
};

export default Footer;
