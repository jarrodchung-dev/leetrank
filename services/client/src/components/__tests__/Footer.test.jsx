import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import Footer from "../Footer.jsx";

describe("<Footer />", () => {
  it("renders properly", () => {
    const wrapper = shallow(<Footer />);
    const span = wrapper.find("span");
    expect(span.text()).toBe("Copyright Â© 2020 Jarrod Chung");
  });

  it("renders a snapshot successfully", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
