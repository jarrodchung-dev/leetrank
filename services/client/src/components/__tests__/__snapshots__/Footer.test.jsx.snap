// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders a snapshot successfully 1`] = `
<footer
  className="footer is-light is-fixed-bottom"
>
  <div
    className="container"
  >
    <div
      className="columns is-multiline"
    >
      <div
        className="column is-4"
      >
        <span>
          Copyright Â© 2020 Jarrod Chung
        </span>
      </div>
      <div
        className="column is-4"
      >
        <div
          className="row"
        />
      </div>
      <div
        className="column is-4"
      />
    </div>
  </div>
</footer>
`;
